torch/_inductor/ir.py

```diff
@@ -6503,9 +6508,16 @@ def export_extern_kernel_node(self):  # type: ignore[no-untyped-def]
         assert isinstance(self, FallbackKernel)
         args, kwargs = self.unflatten_args(self.inputs, self.constant_args)
         args = self.fill_non_provided_args(args, kwargs)
+
+        # Supply any needed kwarg default values.  Any args with default values are
+        # already correctly set thanks to fill_non_provided_args.
+        kwarg_to_value = {
```

Comment:
Does `get_kwargs_value` work here?

Reply:
lol I didn't notice that function. Thanks for the pointer; I'll swap it in!